api/src/peoplemeasurement/views.py

```diff
@@ -67,11 +67,6 @@ def create(self, request, *args, **kwargs):
         try:
             response = super().create(request, *args, **kwargs)
             return response
-        except exceptions.ValidationError as e:
+        except (exceptions.ValidationError, KeyError, TypeError) as e:
             logger.error(f"{e} in message: {request.data}")
```

Comment:
Deze change maakt mijn eerdere change ongedaan. In de globale exception handler heb ik namelijk de error message aangepast, en de `request.data` er niet meer ingezet. Deze is al zichtbaar in Sentry. Doordat hij in de error message stond krijg je steeds unieke errors, wat onhandig is voor reporting.

Reply:
Concrete todo: regel 71 aanpassen naar:

logger.error(e)

Reply:
Edit: dit is gefixed in een latere PR.